core/src/main/java/org/mobilitydata/gtfsvalidator/annotation/FieldTypeEnum.java

```diff
@@ -0,0 +1,21 @@
+package org.mobilitydata.gtfsvalidator.annotation;
+
+public enum FieldTypeEnum {
+    INTEGER,
```

Comment:
In the spec under "Field Types" there is a "non-negative integer" type - do you think that should be explicitly defined here?

Reply:
Would it make sense to add a comment that sets up expectation about how to use INTEGER or FLOAT with regards to negative, non-negative, other potential properties? Also, maybe it makes sense to clarify how these types are related to the official GTFS specification